app/src/main/java/org/schabi/newpipe/error/ErrorUtil.kt

```diff
@@ -92,6 +92,14 @@ class ErrorUtil {
             showSnackbar(fragment, ErrorInfo(throwable, UserAction.UI_ERROR, request))
         }
 
+        /**
+         * Shortcut to calling [Toast.makeText]
+         */
+        @JvmStatic
+        fun showErrorToast(context: Context, message: String) {
+            Toast.makeText(context, message, Toast.LENGTH_LONG).show()
+        }
+
```

Comment:
I don't think this is needed, using the `Toast` methods by themselves in classes which need them ensures that you apply the duration show you wanted.

This is a personal opinion, though.